Modules/Loadable/SubjectHierarchy/Testing/Python/SubjectHierarchyGenericSelfTest.py

```diff
@@ -546,31 +546,31 @@ def testAttributeFilters(filteredObject, proxyModel, showEmptyHierarchyItems=Tru
 
             # Check include node attribute name filter
             self.assertEqual(shProxyModel.acceptedItemCount(shNode.GetSceneItemID()), 9)
-            filteredObject.includeNodeAttributeNamesFilter = ["Markups.MovingInSliceView"]
+            filteredObject.includeNodeAttributeNamesFilter = [slicer.vtkMRMLMarkupsNode.GetMovingInSliceViewAttributeName()]
```

Comment:
I think these tests could be done with any string. @cpinter was there any specific reason to use the `Markups.MovingInSliceView` string as attribute name here?

Reply:
I think I used this because it was easier to test against an attribute that was easy to change with simple interactions. So no.